### DISCUSSION TITLE
How to Change the total % of a Pie instead of 100% ?

### DISCUSSION THREAD

**Discussion by vipindigiqt** [recharts/recharts#2860] [Q&A]
I want to draw a pie for which I only want to render the pie in a fixed percentage value.

For example, only in half of my pie, that is 50% & 50% will be completely empty. (maybe some gray fill)

![image](https://user-images.githubusercontent.com/81612126/170946614-5a3268cd-cee0-4716-8b92-693880bf34a8.png)

Something like in the above image, but I've included the data (double the numbers ).

**Comment by nikolasrieble** [comment]
Does this answer your question? https://recharts.org/en-US/examples/StraightAnglePieChart
